src/terms/xc.jl

```diff
@@ -49,7 +49,7 @@ end
 
     model    = basis.model
     n_spin   = model.n_spin_components
-    @assert all(family(xc) in (:lda, :gga, :mgga, :mggal) for xc in term.functionals)
+    @assert all(family(xc) in (:lda, :hyb_lda, :gga, :hyb_gga, :mgga, :hyb_mgga, :mggal) for xc in term.functionals)
```

Comment:
:hyb_mggal missing

Reply:
See my comments in https://github.com/JuliaMolSim/DftFunctionals.jl/pull/8, maybe remove the `:hyb` here alltogether.